tests/app/service/test_rest.py

```diff
@@ -1,4 +1,5 @@
 import json
```

Comment:
For this test file I see that you can just mock the Zendesk client and assert it was called with the expected arguments. The other file already makes sure that HTTP requests are sent as expected.

Reply:
modified tests to just mock the method `send_create_service`